packages/synapse-react-client/src/lib/containers/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DownloadDocumentTemplate.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, Typography } from '@mui/material'
+import DirectDownloadButton from '../../DirectDownloadButton'
+import { FileHandleAssociation } from '../../../utils/synapseTypes'
+import IconSvg from '../../IconSvg'
+import React from 'react'
+import { useGetFileBatch } from '../../../utils/hooks/SynapseAPI/file/useFiles'
+
+export type DownloadDocumentTemplateProps = {
+  title: React.ReactNode
+  description: React.ReactNode
+  fileHandleAssociation: FileHandleAssociation
+  downloadButtonText: string
+}
+
+export default function DownloadDocumentTemplate(
```

Comment:
Generic version of "Download DUC Template" section, since it's likely we'll have a generic feature at some point.